packages/studio-web/src/app/app.component.html

```diff
@@ -3,7 +3,37 @@
     ReadAlong Studio
   </span>
   <span class="nav-spacer"></span>
-  <div class="nav__buttons">
+  <button class="d-md-none" mat-button [matMenuTriggerFor]="menu">
+    <mat-icon>menu</mat-icon>
+  </button>
+  <mat-menu #menu="matMenu">
+    <button
+      i18n="Studio"
+      mat-button
+      [color]="currentURL === '/' ? 'accent' : ''"
+      [routerLink]="''"
+    >
+      Studio
```

Comment:
You can add the home icon here. So that the alignment looks good in the mobile view
![image](https://github.com/user-attachments/assets/67199691-3c76-431e-88c8-6e4c8bb08f6e)

Reply:
That's an excellent idea. Done, please re-review.